src/components/presentational/LabResultWithSparkline/LabResultWithSparkline.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import LabResultWithSparkline, { LabResultWithSparklineProps } from "./LabResultWithSparkline"
+import Card from "../Card"
+import Layout from "../Layout"
+
+export default {
+    title: "Presentational/LabResultWithSparkline",
+    component: LabResultWithSparkline,
+    parameters: {
+        layout: 'fullscreen',
+    }
+} as ComponentMeta<typeof LabResultWithSparkline>;
+
+const Template: ComponentStory<typeof LabResultWithSparkline> = (args: LabResultWithSparklineProps) =>
+    <Layout>
+        <Card>
+            <LabResultWithSparkline {...args} />
+        </Card>
+    </Layout>;
+
+export const Default = Template.bind({});
+Default.args = {
```

Comment:
I think this needs a definition for `onViewTermInfo(...)`.  At present, clicking the info icon in the storybook causes a JS error.